kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/http/Interceptor.java

```diff
@@ -84,4 +97,15 @@ default CompletableFuture<Boolean> afterFailure(HttpRequest.Builder builder, Htt
     return afterFailure((BasicBuilder) builder, response, tags);
   }
 
+  /**
+   * Called after a connection attempt fails.
+   * <p>
+   * This method will be invoked on each failed connection attempt so can be invoked multiple times for the same request.ID.
```

Comment:
I'm not sure if this information might end up being misleading.
What's the motivation to add it?

What I mean is that users (who don't read carefully) might assume the opposite (for a given operation, for each connection failure, request id will be the same) which isn't true.

Reply:
Yeah I think its so I've removed it. 

While trying to update it I attempted to write a test that caused an unclean disconnect from a websocket connection but wasn't able to. If you think thats important to have do you have any thoughts on how I might trigger it?